added css

tFNiYaFF opened the pull request:
Как-то все очень грустно.

tFNiYaFF committed:
added css

tFNiYaFF committed:
dlc after lecture

DmitryMakhnev commented on style/elements/todo-action_bar.css:
```diff
@@ -0,0 +1,16 @@
+.todos-actions-bar{
+	font-family: Arial;
+    color: #888888;
+    font-size: 10pt;
```
Используем px

DmitryMakhnev commented on style/elements/todo-action_bar.css:
```diff
@@ -0,0 +1,16 @@
+.todos-actions-bar{
+	font-family: Arial;
```
Поправить фонты согласно разбору ДЗ

DmitryMakhnev commented on style/elements/todo-action_bar.css:
```diff
@@ -0,0 +1,16 @@
+    color: #888888;
+    font-size: 10pt;
+	position:relative;
+	width:540px;
```
Ширина должна быть резиновая

DmitryMakhnev commented on style/elements/todo-checkbox.css:
```diff
@@ -0,0 +1,35 @@
+    display:none;            
+}
+
+.todo-item_ready-mark  + label{
```
Очень ненадёжно и неправильно.
Должен быть компонент input-checkbox, см разбор дз

tFNiYaFF replied:
@DmitryMakhnev Но ведь чекбокс в виде input есть, он просто невидимый. На лекции вы показывали с помощью дивов, почему дивами лучше?

DmitryMakhnev commented on style/elements/todo-main.css:
```diff
@@ -0,0 +1,42 @@
+  margin-left:-280px;
+  z-index:3;
+}
+.todos-footer_first{
```
Сделать псевдоэлементами

DmitryMakhnev reviewed (commented):
Сделать согласно разбору дз

tFNiYaFF committed:
DZ2

Добавлен препроцессор

tFNiYaFF committed:
DZ21

Забыл залить файлы

DmitryMakhnev commented on package.json:
```diff
@@ -0,0 +1,25 @@
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:ok-technopolis/preprocessor-stub.git"
```
Вроде у вас другое имя проекта :)

DmitryMakhnev commented on src/styles/project/action-bar/todo-action_bar.scss:
```diff
@@ -0,0 +1,18 @@
+	font-family: $font-non-standart,$font-standart;
+    color: $c-custom-gray;
+    font-size: $fs-xx-small;
+	position:relative;
```
4 строчки === 2 стиля отступа(

DmitryMakhnev commented on src/styles/project/action-bar/todo-action_bar.scss:
```diff
@@ -0,0 +1,18 @@
+@import "../../config/globals.scss";
+
+.todos-actions-bar{
+	font-family: $font-non-standart,$font-standart;
```
Переменную шрифта лучше сделать одну. Она может быть составной

DmitryMakhnev commented on src/styles/project/action-bar/todo-action_bar.scss:
```diff
@@ -0,0 +1,18 @@
+    color: $c-custom-gray;
+    font-size: $fs-xx-small;
+	position:relative;
+	width:540px;
```
Ширина должна быть резиновая

DmitryMakhnev commented on src/styles/project/action-bar/todo-action_bar.scss:
```diff
@@ -0,0 +1,18 @@
+	position:relative;
+	width:540px;
+	bottom:0px;
+	padding:10px;
```
Паддинги переменными

DmitryMakhnev commented on src/styles/project/action-bar/todo-action_bar.scss:
```diff
@@ -0,0 +1,18 @@
+	
+}
+.todos-action-bar_counter{
+	position:absolute;
```
Формирование лайаута на флоатах

DmitryMakhnev reviewed (commented):
Очень мало кода. Пожалуйста ознакомьтесь с разбором дз

tFNiYaFF committed:
New

DZ2

DmitryMakhnev commented on src/styles/project/action-bar/todo-action_bar.scss:
```diff
@@ -1,18 +1,17 @@
-	width:540px;
 	bottom:0px;
-	padding:10px;
+	padding: $offset*2.5;
```
нужно пользоваться стандартными пресетами `$offset_2x` например

DmitryMakhnev commented on package.json:
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "preprocessor-stub",
```
Ваш проект называется по другому

DmitryMakhnev commented on public/index.html:
```diff
@@ -0,0 +1,52 @@
+    <link rel="stylesheet" type="text/css" href="./dist/styles/default.css">
+    <title>TODO</title>
+</head>
+<body>
```
Ховер класс?

DmitryMakhnev commented on public/index.html:
```diff
@@ -0,0 +1,52 @@
+	</div>
+        <div class="todos-list">
+            <div class="todo-item __ready">
+                <input type="checkbox" id="todo-custom-checkbox" checked="checked" class="todo-item_ready-mark" aria-label="Пометить как невыполненное" hidden>
```
Смотрите разбор дз

DmitryMakhnev commented on public/index.html:
```diff
@@ -0,0 +1,52 @@
+    </div>
+	<div class="todos-main">
+    <div class="todos-add">
+            <button class="todo-add_select-all" aria-label="Отметить все как выполненные"></button> 
```
Смотрите разбор дз

DmitryMakhnev commented on public/index.html:
```diff
@@ -0,0 +1,52 @@
+                <input type="checkbox" id="todo-custom-checkbox" checked="checked" class="todo-item_ready-mark" aria-label="Пометить как невыполненное" hidden>
+				<label for="todo-custom-checkbox"></label>
+                <div contenteditable="true" class="todo-item_text">Something</div>
+                <button class="todo-item_remove" aria-label="Удалить todo"></button>
```
Тоже надо смотреть разбор дз

DmitryMakhnev commented on src/styles/base/title/title.scss:
```diff
@@ -0,0 +1,10 @@
+    margin: $offset_2x;
+    padding: $offset;
+    // test
+    background: url('../../../images/favorites-off.png');
```
Тестирвоать хорошо, но при пулл реквесте всё лишнее надо сносить

DmitryMakhnev commented on src/styles/project/button/todo-buttons.scss:
```diff
@@ -0,0 +1,56 @@
+@import "../../config/globals.scss";
+
+.todos-filter{
```
Почему в кнопках описаны фильтры? O_O

DmitryMakhnev commented on src/styles/project/button/todo-buttons.scss:
```diff
@@ -0,0 +1,56 @@
+	cursor:pointer;
+	padding-right:10px;
+}
+.todos-filter.__active{
```
+ всё в разнобой

DmitryMakhnev commented on src/styles/project/checkbox/todo-checkbox.scss:
```diff
@@ -0,0 +1,37 @@
+    display:none;            
+}
+
+.todo-item_ready-mark  + label{
```
Нужно избегать выборок по тегам

DmitryMakhnev commented on src/styles/project/checkbox/todo-checkbox.scss:
```diff
@@ -0,0 +1,37 @@
+@import "../../config/globals.scss";
+
+input[type="checkbox"]{
```
Зачем скрывать все чекбоксы в приложении? O_o

DmitryMakhnev commented on src/styles/project/checkbox/todo-checkbox.scss:
```diff
@@ -0,0 +1,37 @@
+  float:left;
+  cursor:pointer;
+}
+.todo-item_ready-mark:not(checked) + label:after {
```
Дикие не нужные селекторы, пожалуйста ознакомьтесь с разбором дз

DmitryMakhnev reviewed (commented):
Не стал досматривать до конца, очень много проблем(
Пожалуйста пройдите ещё пару раз разбор дз и лекции по css
Так же ознакомьтесь с моей реализацией дз, если будет не понятно по разборам.
Особое внимание уделите разделению на компоненты

tFNiYaFF committed:
Сдаюсь :(

К сожалению совсем уже нет времени разобраться до конца со всем этим и
сделать по человечески :( .
JS тоже пробовал писать, даже работает, но сделано не так, как надо и на
кривом html, который тоже надо править..
Поэтому копирую ваш репозиторий (вы вроде бы разрешили такой вариант на
крайний случай, если это мне не приснилось, конечно). Прошу понять и
простить и поставить баллов на троечку :(.

tFNiYaFF committed:
DZ

DmitryMakhnev reviewed (commented):
Если копировать, то давайте тогда и стили нормально переносить.
